style.css

```diff
@@ -0,0 +1,271 @@
+*{
+    margin:0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: sans-serif;
+}
+:root{
+    --primary-color:#424874;
+    --secondary-color:#a6b1e1;
+    --txt-primary:#dcd6f7;
+    --txt-secondary:#d1d4e4;
+    --txt-dark:#424874;
+    --txt-light:#a6b1e1;
+    --padding-sm:0.5rem;
+    --padding-md:1rem;
+    --padding-lg:1.5rem;
+}
+html {
+    scroll-behavior: smooth;
+    overflow-x: hidden;
+}
+body{
+    width: 100vw;
+    min-height: 100vh;
+    background: #f7f7f7;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    overflow-x: hidden;
+}
+a{
+    text-decoration: none;
+    color: inherit;
+}
+p{
+    line-height: 1.8rem;
+}
+h1{
+    margin:1rem 0;
+}
+.header-index{
+    width: 100vw;
+    padding: 1rem 1rem;
+    position: sticky;
+    top:0;
+    z-index: 2;
+    background: #f7f7f7;
+}
+.header{
+    width: 100vw;
+    padding: 1rem 1rem;
+    position: sticky;
+    top:0;
+    z-index: 2;
+    background: #f7f7f7;
+    display: none;
+}
+.brand-link{
+    color:var(--primary-color);
+    font-size: 1.2rem;
+}
+.brand-name{
+    display: inline-flex;
+    align-items: center;
+    width:max-content;
+    cursor:pointer;
+}
+.brand-logo{
+    width:3rem;
+    margin-right: 0.5rem;
+}
+main{
+    margin: auto 0;
+    width: 75%;
+    max-width: 30rem;
+    text-align: center;
+}
+main > .brand-logo{
+    width: 8rem;
+}
+.brand-tagline{
+    font-size: 1.5rem;
+    color: var(--primary-color);
+    margin:1rem 0;
+    letter-spacing: 0.1rem;
+}
+.btn{
```

Comment:
Hey,
The usage of `!important` can be avoided here as it will override any default styles being applied on `.btn` class.
I think it would be better if you give a specific class for adding `padding-top` and `padding-bottom`.

Reply:
Actually this is not the component css, This is my documentation css . The default style I have here was not overriding the component css that's why I had to give !important